groups/sig-release/groups.yaml

```diff
@@ -220,3 +220,35 @@ groups:
       - saschagrunert@gmail.com
       - stephen.k8s@agst.us
       - tpepper@vmware.com
+
+  - email-id: release-team@kubernetes.io
+    name: release-team
+    description: |-
+      Release Team communications.
+      Includes SIG Release Leads and current release team.
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      MembersCanPostAsTheGroup: "true"
```

Comment:
Drop this. We don't necessarily want everyone posting as the group.

Reply:
Done.